src/cljs/status_fiddle/react_native_web.cljs

```diff
@@ -1,26 +1,26 @@
 (ns status-fiddle.react-native-web
-  (:require [reagent.core :as reagent]
-            [react-native-web :refer [View]]))
+  (:require [reagent.core :as reagent]))
 
-#_(defonce react-native-web
-    (cond (exists? js/ReactNative) js/ReactNative
-          (exists? js/require) (or (js/require "react-native-web")
-                                   (throw (js/Error. "require('react-native-web') failed")))
-          :else (throw (js/Error. "react-native-web is missing")))
+(defonce react-native-web
+  (cond (exists? js/ReactNative) js/ReactNative
```

Comment:
so you can use it here just `(defonce react-native-web js/ReactNativeWeb)` if you added `window.ReactNativeWeb = window.deps['react-native-web'];`

Reply:
so we don't need this `defonce react-native-web`